tlsutil/config.go

```diff
@@ -561,6 +567,16 @@ func (c *Configurator) VerifyServerHostname() bool {
 	return c.base.VerifyServerHostname || c.autoEncrypt.verifyServerHostname
 }
 
+// IncomingGRPCConfig generates a *tls.Config for incoming GRPC connections.
+func (c *Configurator) IncomingGRPCConfig() *tls.Config {
+	c.log("IncomingGRPCConfig")
```

Comment:
What are these logs used for?

Reply:
I am using them for debugging. They are showing the version of the config that is being used to generate the `*tls.Config`. Eg when you reload consul so that a new cert is being used, this log line should show a version increased by one. If it doesn't increase, something is wrong.